packages/analytics/src/tests/analytics.test.ts

```diff
@@ -57,6 +57,40 @@ describe('analytics', () => {
         expect(global.fetch).toHaveBeenCalledTimes(2);
     });
 
+    it('should report even without environment option', () => {
+        const mockFetchPromise = Promise.resolve({
+            json: () => Promise.resolve({}),
+        });
+        global.fetch = jest.fn().mockImplementation(() => mockFetchPromise);
+        // @ts-expect-error
+        jest.spyOn(global, 'fetch').mockImplementation(() => mockFetchPromise);
+
+        const timestamp = new Date().getTime();
+        jest.spyOn(Date, 'now').mockImplementation(() => timestamp);
+        jest.spyOn(console, 'error').mockImplementation(() => {});
+
+        const app = 'connect';
```

Comment:
according to the `App` type `'suite' | 'connect-popup'`
```suggestion
        const app = 'connect-popup';
```

Reply:
fixed, squashed, rebased